src/Catalog/Helpers/DeletionAuditEntry.cs

```diff
@@ -163,7 +173,7 @@ private void InitValues()
 
             return
                 (await Task.WhenAll(
-                    auditRecords.Select(record => DeletionAuditEntry.Create(auditingStorage, record.Uri, cancellationToken, logger))))
+                    auditRecords.Select(record => DeletionAuditEntry.CreateAsync(auditingStorage, record.Uri, cancellationToken, logger))))
                 // Filter out null records.
```

Comment:
I think that's fine because we probably want to know if there's a corrupt audit log. Nice catch!